docs/guides/vault/interact-with-contracts.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Interact With Contracts'
+sidebar_position: 3
+---
+
+# Interact With Contracts
+
+Similar to our **[previous guide](./edit-vault-data.md)** on setting data on the Vault, in this guide we will be learning how to **interact with other contracts** through the execute function on the Vault.
+
+## Interact with contracts
+
+Check the **previous guide** to know how to get the needed variables.
```

Comment:
Would be good to specify which guide with the link, and also specify which variables are being referred to in 'needed variables'